src/store/slices/playlists/slice.ts

```diff
@@ -35,13 +37,32 @@ string,
         }
 })
 
+const getCurrentUserPlaylists = createAsyncThunk<
+    string,
+    string,
+    { state: RootState } >
+    ('playlists/getCurrentUserPlaylists',
+    async (payload, thunkApi) => {
+        const id = payload;
+        try {
+            const currentUserOwnPlaylists = await firestoreApi.getUserOwnPlayLists(id)
+            thunkApi.dispatch(slice.actions.SET_OWN_PLAYLISTS(currentUserOwnPlaylists))
+            return 'sets own playlists'
```

Comment:
Would be better not to use spaces in these return values consequently, as we will convert many of them to object keys later for the notification.  It would just make our life easier.